Source/WebCore/html/HTMLTextFormControlElement.cpp

```diff
@@ -68,6 +68,10 @@ using namespace HTMLNames;
 
 static Position positionForIndex(TextControlInnerTextElement*, unsigned);
 
+// DEAR REVIEWER: this is done solely to reduce the diff and improve reviewing experience.
```

Comment:
You're not going to add this comment, right? You're actually going to rename it?

Reply:
Yes, I'm absolutely not going to add this comment and will do the rename right before landing.

Reply:
Renamed.